portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -41,6 +59,24 @@ const getFormattedReportPeriod = () => {
   return format(lastMonthDate, 'MMMM yyyy');
 };
 
+/**
+ * Checks if a utilisation report has already been submitted by the specified bank for the current reporting period
+ * @param bank {object} - the bank to check
+ * @returns {Promise<boolean>}
+ */
+const getIsReportSubmitted = async (bank) => {
+  // TODO FN-1164 - check how this endpoint has been updated
+  const reportsResponse = await api.getUtilisationReports(bank.id);
+
+  if (reportsResponse?.status !== 200) {
```

Comment:
I refactored the endpoint to just return data/throw though so this will need to be refactored to account for this